### DISCUSSION TITLE
ArcSDM 5

### DISCUSSION THREAD

**Discussion by tomironkko** [gtkfi/ArcSDM#162] [Announcements]
This release is the last release for ArcMap. ArcSDM 6 will support only ArcGIS Pro. 

Use this release until ArcSDM 6 becomes available.

<hr /><em>This discussion was created from the release <a href='https://github.com/gtkfi/ArcSDM/releases/tag/v5.03.01'>ArcSDM 5</a>.</em>

**Comment by Adilos12345** [comment]
TO TOOL DOESN'T WORK
